joint_limits/include/joint_limits/joint_limits_rosparam.hpp

```diff
@@ -216,7 +284,7 @@ inline bool getJointLimits(
  * \return True if a complete soft limits specification is found (ie. if all \p k_position, \p k_velocity, \p soft_lower_limit and
  * \p soft_upper_limit exist in \p joint_limits/joint_name namespace), false otherwise.
  */
-inline bool getSoftJointLimits(
+inline bool get_joint_limits(
```

Comment:
Shouldn't line 287 be get_soft_joint_limits? Or else it is just declaring the get_joint_limits from line 129 again?

Reply:
I think it is based on the argument you parse, you either get the hard limits or soft limits

Reply:
I don't see a reason to have two differently named methods. What I propose is simple and straightforward.

Reply:
Agreed!

Reply:
Thanks for clarifying. Makes more sense now.